### PR TITLE
Add google/yapf python formatter

### PR DESCRIPTION
**Pull request by kmcnellis**
### Description
Use the https://github.com/google/yapf Google Python styleguide formatter for python code.  Added to bin/fmt, and as a github action on pull requests.

### Checklist
- [ ] Ran it and committed the updates

### CODE REVIEW

**Review by azizova-leyla — CHANGES_REQUESTED** [review]
This is great! 
Can you remove ts files from this PR, they seem unrelated.

**Comment by kmcnellis** [comment]
> This is great! Can you remove ts files from this PR, they seem unrelated.

Those were added by the formatter.  I'm not quite sure why.

**Comment by kmcnellis** [comment]
> > This is great! Can you remove ts files from this PR, they seem unrelated.
> 
> Those were added by the formatter. I'm not quite sure why.

Added https://github.com/seattle-uat/civiform/pull/2703 to format those files

**Review by azizova-leyla — APPROVED** [review]

File: formatter/fmt [hunk 1]
```diff
@@ -45,3 +45,7 @@ echo 'Start format shell'
 shfmt -bn -ci -i 2 -w -l \
   $(shfmt -f . | grep -v -e /node_modules)
 echo 'Done formatting shell'
+
+echo 'Start format python'
+yapf -i -r --no-local-style -p -vv --style='{based_on_style: google, SPLIT_BEFORE_FIRST_ARGUMENT:true}' .
```
Comment (bion) [inline formatter/fmt:50]: suggestion (blocking): line length

**Pull request merged**